src/User.js

```diff
@@ -22,12 +22,12 @@
     this.friendsActivityRecords = []
   }
   getFirstName() {
-    this.name = this.name.split(' ');
-    return this.name[0].toUpperCase();
+    const newName = this.name.split(' ');
```

Comment:
Good fix for our issue where the user.name property was changing to an array every time this method was called.